### PR TITLE
support blocked format for broadcast

### PR DESCRIPTION
**Pull request by si-eun-kim**
### Details:
 - supports all blocked format
 - test cases added

### Tickets:
 - 74956

### CODE REVIEW

**Comment by yeonbok** [comment]
Please add fusion tests

File: src/plugins/intel_gpu/src/graph/gather_elements.cpp [hunk 1]
```diff
@@ -24,11 +24,12 @@ layout gather_elements_inst::calc_output_layout(gather_elements_node const& node
     auto input_layout = input_layout_origin.size.sizes(input_layout_origin.format);
     auto indices_layout = indices_layout_origin.size.sizes(indices_layout_origin.format);
 
-    if (node.has_fused_primitives()) {
-        input_layout_origin.data_type = node.get_fused_output_layout().data_type;
-    }
+    // if (node.has_fused_primitives()) {
+    //     input_layout_origin.data_type = node.get_fused_output_layout().data_type;
+    // }
```
Comment (yeonbok) [inline src/plugins/intel_gpu/src/graph/gather_elements.cpp:29]: please remove comments

File: src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp [hunk 2]
```diff
@@ -64,6 +64,25 @@ void prepare_primitive_fusing::run(program& p) {
...
+                if (prev->as<reshape>().input().get_output_layout() == node.get_output_layout()) {
+                    p.add_optimized_primitive_info(prev->id());
+                    p.add_optimized_primitive_info(node.id());
+                    p.extract_and_remove(*prev);
+                    p.extract_and_remove(node);
+                }
+            }
+        });
+    }
+
```
Comment (yeonbok) [inline src/plugins/intel_gpu/src/graph/graph_optimizer/prepare_primitive_fusing.cpp:84]: could you merge these two loops ?

File: src/plugins/intel_gpu/tests/test_cases/broadcast_gpu_test.cpp [hunk 3]
```diff
@@ -133,45 +139,148 @@ void start_broadcast_test_5d(data_types cldnn_data_type, std::vector<size_t> out
...
+}
+
+TEST(broadcast_gpu_int8_t, b_fs_yx_fsv16_1_to_5_w_b_axes_0) {
+    std::vector<int8_t> golden_data = {1, 1, 1, 1, 1};
+    start_broadcast_test<int8_t>(format::b_fs_yx_fsv16, data_types::i8, {5}, {1}, {0}, golden_data);
+}
+
+TEST(broadcast_gpu_int8_t, b_fs_yx_fsv32_1_to_5_w_b_axes_0) {
+    std::vector<int8_t> golden_data = {1, 1, 1, 1, 1};
+    start_broadcast_test<int8_t>(format::b_fs_yx_fsv32, data_types::i8, {5}, {1}, {0}, golden_data);
```
Comment (yeonbok) [inline src/plugins/intel_gpu/tests/test_cases/broadcast_gpu_test.cpp:172]: This kind of test is not worthy where the feature size < feature block size. 
Could you make the test shapes for blocked layouts for more meaningful ones and compare them with ngraph result?

**Comment by akladiev** [comment]
This PR will be closed in 2 weeks in case of no activity.

**Comment by akladiev** [comment]
This PR was closed because it has been stalled for 2 week with no activity.

**Pull request closed without merging**